modules/api/php/projectimagesrow.class.inc

```diff
@@ -66,7 +96,7 @@ class ProjectImagesRow implements \LORIS\Data\DataInstance
      */
     public function getCenterID(): int
     {
-        return $this->DBRow['CenterID'];
+        return intval($this->_centerid);
```

Comment:
Just FYI `intval()` will return 0 when the input is null. This might not matter (and I'm not sure how you handle invalid values elsewhere) but this might be important info.

Reply:
I did not know. Thanks.
I this case, getCenterID must return an `int` so `0` is convenient for garbage data. Until someone creates a PSC with ID = 0